### ISSUE TITLE
Should we be moving things to https://github.com/scale-ms?

### ISSUE THREAD

**Issue by mrshirts** [radical-collaboration/SCALE-MS#10]
Was that what we were intending to do with the information now?

**Comment by eirrgang** [comment]
Sounds reasonable to me. I note that there are at least three NSF award numbers, but this repo only has one. Do we want to note all of them?

**Comment by mturilli** [comment]
I do not care where this repository is as far as it is used to progress the project's goals. @eirrgang please feel free to edit the NSF numbers as you see it fit. If you want to migrate the repository, I will see how to do it without loosing all the work we have been doing here already.

**Comment by eirrgang** [comment]
I created https://github.com/SCALE-MS/scale-ms and cloned the https://github.com/SCALE-MS/scale-ms/wiki
I also set up the webhook for https://scale-ms.readthedocs.io/

If we want to finish the migration, we can transfer the issues.
We could transfer this repository to the other organization, but I don't know if that transfers the issues, and the readthedocs web hook would need to be reinstalled, if we want that. (web hook is distinct from content and configuration: https://github.com/SCALE-MS/scale-ms/pull/1 )

If we don't want to do the migration, we could set up a readthedocs webhook for this repository and I could reissue pull requests here.

**Comment by mturilli** [comment]
If the issues can be transferred as they are to the other repository, I would suggest to do the transfer and then close this repository.

**Comment by eirrgang** [comment]
> If the issues can be transferred as they are...

I don't think so. [You can only transfer issues between repositories owned by the same user or organization account.](https://help.github.com/en/github/managing-your-work-on-github/transferring-an-issue-to-another-repository)

SCALE-MS is a different organization.

**Comment by eirrgang** [comment]
I copied and quoted the existing issues, and I duplicated the tags. Issue numbers are different. I found and fixed some cross-linking errors, but there may be others.

https://github.com/SCALE-MS/scale-ms/issues

**Comment by eirrgang** [comment]
It doesn't look like all of the collaborators are yet members of the [GitHub SCALE-MS organization](https://github.com/orgs/SCALE-MS/people) or one of its [teams](https://github.com/SCALE-MS/scale-ms/settings/access), though we did get everyone from [here](https://github.com/orgs/radical-collaboration/teams/scale-ms/members)
